server.js

```diff
@@ -56,13 +85,15 @@ app.delete('/api/v1/vineyards/:id', (request, response) => {
   //also make sure to run the delete all wines code first
 });
 
-app.get('/api/v1/vineyards', (request, response) => {
-  // will be used for query
-  //?= query for searching each region, name, 
-  //return 200
-  //return 404
-  //return 500
-});
+// app.get('/api/v1/vineyards', (request, response) => {
```

Comment:
Lets review need for this to be commented out

Reply:
removed unnecessary commented out code